tst/EnergyPlus/api/test_OutputFiles.py

```diff
@@ -55,13 +55,23 @@
 
 import glob as gb
 import os
-import shlex
 import tempfile
 import unittest
 
 from pyenergyplus.api import EnergyPlusAPI
 
 
+def parent(x):
+    return os.path.dirname(os.path.realpath(x))
+
+
+this_file = os.path.realpath(__file__)  # returns C:\EPRepo\tst\EnergyPlus\api\test_OutputFiles.py
+repo_root = parent(parent(parent(parent(this_file))))  # returns C:\EPRepo
+# something **very** odd was happening with os.path.join here on Windows.  Putting a couple separators manually works OK
+idf_path = repo_root + os.sep + os.sep + "testfiles" + os.sep + os.sep + "1ZoneUncontrolled.idf"
```

Comment:
@Myoldmopar that looks fishy. But it's a test, so whatever.

Reply:
also, before there was a configure step via cmake with `@IDF_FILE@`. Are you sure you wanted to harcode it to 1ZoneUncontrolled.idf?

Reply:
I do, I like being able to execute the Python script from outside the context of ctest, and having this @ in there requires CMake to configure it first.

Reply:
And yeah, it was super fishy -- I definitely would not do this in production code, but in the test I'm willing to leave this in place as it works.